src/app/invitation-message/blue-bar/invitation-message-blue-bar.controller.js

```diff
@@ -33,6 +33,8 @@ function calInboxInvitationMessageBlueBarController(
   self.onPartstatChangeError = onPartstatChangeError;
   self.isActionable = isActionable;
   self.translationData = {};
+  self.isOutdated = false;
+  self.outdatedEventMessage = 'Event outdated';
```

Comment:
```suggestion
  const outdatedEventMessage = 'Event outdated';
```
no need to bind this to the controller